database/migrations/2017_07_27_082531_create_keyword_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateKeywordTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('wt_keyword', function($table){
```

Comment:
tên bảng keyword anh đồng ý đi để các e k phải sửa.Nhưng mà tên trường các e đặt kiểu này anh k đồng ý đc.
keyword_id -> id, value->keyword, not null các trường a chưa có thấy, PK của bảng anh cũng chưa thấy .....